submissions/tatiana-jess/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="styles/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <!-- Code Fellows favicon -->
+    <link rel="icon" type="image/x-icon" href="https://i.imgur.com/7v5ASc8.png">
+  </head>
+  <body>
+    <header class="site-header clearfix">
+      <section class="heading-group-h1">
+        <h1>Kilovolt Blog!</h1>
+        <h2>The shocking power of static pages</h2>
+      </section>
+      <nav class="main-nav">
+        <!-- Hamburger Menu -->
+        <div class="icon-menu"></div>
+        <ul>
+          <li class="tab" data-content="articles"><a href="#" class="icon-home">Home</a></li>
+          <li class="tab" data-content="about"><a href="#" class="icon-address-book">About (coming soon!)</a></li>
+          <li><a href="http://twitter.com/codefellowsSEA" target="_blank" class="icon-twitter3">Twitter</a></li>
+          <li><a href="http://github.com/" target="_blank" class="icon-github">Github</a></li>
+        </ul>
+      </nav>
+    </header>
+      <main>
+        <section id="articles">
+          <article class="template" data-category="cat">
+            <header>
+              <h1>Title</h1>
+              <div class="byline">
+                By <address><a href="">Author Name</a></address>
+                published <time pubdate datetime="2000-01-01">Publish Time</time>
+              </div>
+            </header>
+            <section class="article-body"></section>
+          </article>
+        </section>
+      </div>
+    </main>
+    <footer>
+      &copy; 2016 Code Fellows | Static Blog | Happy Ipsum!
+    </footer>
+    <!-- // TODO: Include any/all needed script tags -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script src="scripts/blogArticles.js"></script>
+    <script src="scripts/article.js"></script>
```

Comment:
Good job here recognizing the importance of script loading 👍 
We say that this is a matter of dependency management in that we have certain files that rely on others' variables to be available AKA attached to the window object.